inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_sequences_to_sequences_ie.cpp

```diff
@@ -146,27 +134,23 @@ ngraph::pass::ConvertRNNSequenceMatcher::ConvertRNNSequenceMatcher() {
             return false;
         }
 
-        auto W = std::dynamic_pointer_cast<ngraph::opset4::Constant>(
-                rnn_sequence->input_value(3).get_node_shared_ptr());
-        if (!W) {
+        // Bidirectional cases are not supported
+        // todo: add exception
```

Comment:
TODO? Why cannot you add exception in this PR?

Reply:
I don't see any exceptions in any transformation. As I understand the logic of this: if the transformation was not successful (return false), then theoretically there is an opportunity to apply another transformation or other way to manage this situation, but exception broke this.

Reply:
In this case please remove todo. From my point of view it is OK, that we return false in case of error.
We shouldn't throw exception in this case.

Reply:
fixed